### PR TITLE
docs(borealis): DOC-455 add explanation for fields

### PR DESCRIPTION
**Pull request by 317brian**
* Adds explanation for all the fields in the YAML

### CODE REVIEW

File: content/en/borealis/Reference/ref-deployment-file.md [hunk 1]
```diff
@@ -41,15 +41,15 @@ targets:
     strategy: <strategyName>
 # The list of manifests sources
 manifests:
-  # A directory containing multiple manifests. Instructs Borealis to read all yaml|yml files in the directory and deploy all manifests to the target defined in    `targets`.
+  # A directory containing multiple manifests. Borealis all yaml|yml files in the directory and deploy all manifests to the target defined in    `targets`.
```
Comment (bbrina) [inline content/en/borealis/Reference/ref-deployment-file.md]: I'm not sure if this is 100% understandable, might be missing some words

File: content/en/borealis/Reference/ref-deployment-file.md [hunk 2]
```diff
@@ -152,59 +199,126 @@ The path to a manifest file that you want to deploy or the directory where your
...
 Borealis progresses through all the steps you define as part of the deployment process. The process is sequential and steps can either be of the type `setWeight` or `pause`.
 
 Generally, you want to configure a `setWeight` step and have a `pause` step follow it although this is not necessarily required.
 
-Here are two scenarios where this pairing sequence might not be used:
+Some scenarios where this pairing sequence might not be used would be the following:
 
-- You can start the sequence of steps with a `pause` that has no corresponding weight. That behaves as though the weight is `0` since it is as the start of the deployment. This results in the pause step happening before any of the app is deployed.
-- You can follow a `setWeight` step with two `pause` steps. For example, after hitting the weight threshold, you could add a `pause` that waits for a certain amount of time and a pause that waits for a manual judgment.
+- You can start the sequence of steps with a `pause` that has no corresponding weight. This behaves as though the weight is `0` since it is as the start of the deployment. This causes the deployment to pause at the start before any of the app is deployed.
```
Comment (bbrina) [inline content/en/borealis/Reference/ref-deployment-file.md]: it is at

File: content/en/borealis/Reference/ref-deployment-file.md [hunk 3]
```diff
@@ -152,59 +199,126 @@ The path to a manifest file that you want to deploy or the directory where your
...
+- You want to have two `pause` steps in a row, such as a `pause` for a set amount of time followed by a `pause` for a manual judgment.
 
 You can add as many steps as you need but do not need to add a final step that deploys the app to 100% of the cluster. Borealis automatically does that after completing the final step you define.
 
 ### `strategies.<strategyName>.<strategy>.steps.setWeight.weight`
 
-This is an integer value and determines how much of the cluster the app gets deployed to. The value must be between 0 and 100 and the the `weight` for each `setWeight` step should increase as the deployment progresses. After hitting this threshold, Borealis  pauses the deployment based on the behavior you set for `strategies.<strategyName>.<strategy>.steps.pause`.
+This is an integer value and determines how much of the cluster the app gets deployed to. The value must be between 0 and 100 and the the `weight` for each `setWeight` step should increase as the deployment progresses. After hitting this threshold, Borealis  pauses the deployment based on the behavior you set for  the `strategies.<strategyName>.<strategy>.steps.pause` that follows.
```
Comment (bbrina) [inline content/en/borealis/Reference/ref-deployment-file.md]: Double space after Borealis

File: content/en/borealis/Reference/ref-deployment-file.md (hunk 1 above)
Comment (317brian) [reply content/en/borealis/Reference/ref-deployment-file.md]: Haha, it is 

```suggestion
  # A directory containing multiple manifests. Borealis reads all yaml|yml files in the directory and deploys the manifests to the target defined in `targets`.
```

**Comment by 317brian** [comment]
Ivan provided feedback on Slack.

**Pull request merged**